SPDY/SPDYOriginEndpointManager.h

```diff
@@ -17,6 +19,8 @@
 @property (nonatomic, readonly) SPDYOrigin *origin;
 @property (nonatomic, readonly) SPDYOriginEndpoint *endpoint;
 @property (nonatomic, readonly) NSUInteger remaining;
+@property (nonatomic, readonly) SPDYProxyStatus proxyStatus;
+@property (nonatomic) BOOL authRequired;  // writable since only the socket knows the answer
```

Comment:
For consistency let's continue to use the bool type internally and only use BOOL for public interfaces.